index.js

```diff
@@ -0,0 +1,7 @@
+const menu = document.querySelector('.menu-bars');
+const nav = document.querySelector('.nav-links');
+
+menu.addEventListener('click', () => {
+  nav.classList.toggle('nav-active');
+  console.log(nav);
+});
```

Comment:
Please complete the logic of the function as it's not opening when I click on the menu bars
It should show the nav links when I click on the menu bars
Also, you should not include the debugging code (console.log)